src/Rule/Boolean.php

```diff
@@ -39,9 +39,6 @@ class Boolean extends Rule
      */
     public function validate($value)
     {
-        if ($value === true || $value === false) {
-            return true;
-        }
-        return $this->error(self::NOT_BOOL);
+        return is_bool($value) ?: $this->error(self::NOT_BOOL);
```

Comment:
Oh I'm sorry for not being clear enough, that I absolutely agree with you about. but I think this would be easier to read:

```php
if (is_bool($value) {
    return true;
}

return $this->error(self::NOT_BOOL);
```